Sources/XiCliCore/CommandLineTool.swift

```diff
@@ -110,12 +116,12 @@ public final class CommandLineTool {
     func help() {
         let message = """
                 The Xi CLI Help:
-                xi <file> [--wait | -W] [--help | -h]
+                xi <file> [--wait | -w] [--help | -h]
```

Comment:
```suggestion
                xi <file>... [--wait | -w] [--help | -h]
```
This just brings it in line with [the docs](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/cc771080(v=ws.11)) used for the rest of the help